src/main/java/io/vertx/ext/mongo/AggregateOptions.java

```diff
@@ -25,20 +25,19 @@
    */
   public static final long DEFAULT_MAX_AWAIT_TIME = 1000L;
 
-  private int     batchSize;
-  private long    maxTime;
-  private long    maxAwaitTime;
+  private int batchSize;
+  private long maxTime;
+  private final long maxAwaitTime;
```

Comment:
A data object field should not be `final`.

Reply:
It's not going to be changed, since there is no getter/setter for it. I can remove it, still it is de facto final